checkstyle.xml

```diff
@@ -146,7 +146,7 @@
         <module name="EqualsHashCode"/>
         <!--<module name="HiddenField"/>-->
         <module name="IllegalInstantiation"/>
-        <module name="InnerAssignment"/>
+        <!-- <module name="InnerAssignment"/>-->
```

Comment:
Hey @smic1909 

What's the reasoning between commenting this check ?

Reply:
Good catch. This was needed for some changes I did locally. Reverted.